src/components/test/common-tests.js

```diff
@@ -25,22 +37,23 @@ const createComponent = (Component, props = {}) => {
  */
 export function testCompositeComponent(
   Component,
-  { componentName, elementSelector } = {}
+  { componentName, content, elementSelector } = {}
 ) {
   const displayName = componentName ?? Component.displayName ?? Component.name;
+  const contentFn = content ?? createComponent;
```

Comment:
This could use default assignment in destructuring (`{componentName, content = createComponent, ...}`), to eliminate the `contentFn` variable.